pandas/core/series.py

```diff
@@ -158,6 +158,8 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
 
             if data is None:
                 data = {}
+                if index is None:
```

Comment:
you prob just need to remove the ``check_index_type=False`` and then your change should work